### PR TITLE
Glob_Patterns.rst: Link to .coafile options

### PR DESCRIPTION
**Pull request by yash-nisar**
Fixes https://github.com/coala/documentation/issues/73

### CODE REVIEW

File: Users/Glob_Patterns.rst [hunk 1]
```diff
@@ -40,7 +40,9 @@ The special characters used in shell-style wildcards are:
 .. note::
 
     If you're looking for a negation pattern to exclude paths, check out the
-    ``--ignore`` argument or ``ignore`` .coafile option!
+    ``--ignore`` argument or ``ignore`` .coafile option `here
+    <http://coala.readthedocs.io/en/latest/Users/Tutorial.html#ignoring
```
Comment (yash-nisar) [inline Users/Glob_Patterns.rst]: @Adrianzatreanu Why is there a new line after `here` above ? My vim editor shows properly indented code here : 
glob_pattern
Is the P.R ready to be merged ?

Comment (adtac) [inline Users/Glob_Patterns.rst]: They're separate lines in your editor too (lines 43 and 44)

**Comment by adtac** [comment]
ack https://github.com/coala/documentation/pull/160/commits/b358024fefc22e2082267308a22e87917f29d357

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**